src/components/TableFilter.js

```diff
@@ -274,13 +274,15 @@ class TableFilter extends React.Component {
       (column.filterOptions && column.filterOptions.display) ||
       (options.filterOptions && options.filterOptions.display);
 
+    if (!display) {
+      console.log('Property "display" is required when using custom filter type.');
```

Comment:
Can we use `console.error` here so that it will be more clear to the devs that they forgot something?

Reply:
Updated!